assets/readme/vuedotjs.svg

```diff
@@ -0,0 +1,4 @@
+<svg fill="white" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please minimize the icons. Also, why do you've changed the names from `<icon>-white` to `<icon>`? It seems to me more understandable with `-white` because are the white versions.

Reply:
Icons are taken from the icons folder as is. If the icons are not minimized here the only thing I can think of is that they are not minimized there.
There is no need for white since these are the only versions in assets/readme, and it makes comparison much easier in test.

Reply:
> There is no need for white since these are the only versions in assets/readme, and it makes comparison much easier in test.

Sorry, I don't agree. How makes more easy for tests the addition of `-white` to the files? File names must express their meaning.